src/app/components/layout/navbar/navbar.component.css

```diff
@@ -86,6 +86,13 @@
   font-size: 0.7rem;
 }
 
-.default-cursor {
-  cursor: default;
-}
+.disabled {
+  opacity: .95;
+  background-color: whitesmoke;
+  pointer-events: none;
+ }
+
+ .help {
+  margin-top: 0px;
```

Comment:
Not for any with help class but for this ID only?!

Reply:
help class is used only in the span that is to show the Auto refresh not enable message in navbar.